### PR TITLE
[sil_nkoo] New Desktop and mobile keyboard

### LINKED ISSUE #1291: [n'ko] User requests touch layout keyboard
One of the Keyman users corresponding on the Keyman [Facebook page](https://www.facebook.com/KeymanApp/) has asked for help in creating a lexical-model and touch layout keyboard for N'ko.

He's sent me a wordlist to create the lexical-model, and also asked 

> S'il vous plaît, si vous pouvez reorganiser le clavier N'ko au disposition de cette image, parce-que c'est ça le standard de nôtres clavier N'ko:

(Google Translate)
> Please, if you can rearrange the N'ko keyboard in the layout of this image, because that is the standard of our N'ko keyboard:

### Default layer
default layer

### Shift layer
shift layer

# Numeric layer
number layer

### Legacy keyboard
I see there's legacy/n/nko keyboard, but the keyboard package currently doesn't include the touch layout .js file.

### PR DESCRIPTION
**Pull request by LornaSIL**
I used the phone layout created by @darcywong00, and created a new desktop layout based on a keyboard layout I found that resembled the mobile screenshots.

* After using the phone layout I then added in all the other characters that were on the desktop keyboard.
* I had trouble testing on my phone with the Noto Sans Nko font. I had to switch to Ebrima (which we cannot distribute)

Fixes #1291

### CODE REVIEW

**Comment by darcywong00** [comment]
I won't be available to review until I get back July 27

**Comment by darcywong00** [comment]
The Noto font lacks 3 characters from Unicode 11 (on the shift layer of the touch keyboard)
See https://github.com/googlefonts/noto-fonts/issues/1419

File: release/sil/sil_nko/source/sil_nko.keyman-touch-layout [hunk 1]
```diff
@@ -0,0 +1,873 @@
...
+                "width": "110"
+              },
+              {
+                "id": "U_07ED",
+                "text": "\u07ED",
+                "width": "110"
+              },
+              {
+                "id": "U_07EE",
+                "text": "\u07EE\u07EE",
```
Comment (darcywong00) [inline release/sil/sil_nko/source/sil_nko.keyman-touch-layout:433]: I had a stray copy-paste here. Can you remove the duplicate?

**Comment by darcywong00** [comment]
On the desktop OSK
Shift-L can have a keycap, right?

**Comment by LornaSIL** [comment]
Yes, it is displaying for me, but I have a font supporting it.

**Comment by darcywong00** [comment]
Just checking how number keys usually go on RTL keyboards.

On the touch layout, the number keys run (left to right) 0, 9, 8, ... 1
I see the desktop OSK follows that layout, but does that mean K_1 outputs NKO_DIGIT_ZERO and K_0 outputs NKO_DIGIT_ONE?

**Comment by LornaSIL** [comment]
Yes, that was extremely confusing to me,  but the other Nko layouts that I
looked at online seemed to follow that behavior. Arabic keyboards do not do
that.

**Pull request merged**